README.md

```diff
@@ -3,21 +3,30 @@
 
 # Azure IoT Edge LoRaWAN Starter Kit
 
-The LoRaWAN starter kit is an *experimental* cross platform private network implementation of the [LoRaWAN specification]('https://lora-alliance.org/resource-hub/lorawantm-specification-v102') built for connectivity to Azure IoT Hub. It enables users to setup their own LoRaWAN network that can connect to LoRa based nodes (sensors) and send decoded message packets to Azure IoT Hub for cloud based processing, analytics and other workloads. Alternatively, it allows sending commands from the cloud to the end nodes. The goal of the the project is to provide guidance and a reference for Azure IoT Edge users to experiment with LoRaWAN technology.
+The Azure IoT Edge LoRaWAN starter kit is an *experimental* cross platform LoRaWAN implementation following the [LoRaWAN 1.0.2 specification](https://lora-alliance.org/resource-hub/lorawantm-specification-v102). The goal of the solution is to provide a turnkey experience for users looking to build their own private LoRa network and connect their LoRaWAN devices and gateways to Azure IoT Hub without the need of any intermediaries or third party services. The kit provides a LoRaWAN packet forwarder and network server implementation built on top of [Azure IoT Edge](https://azure.microsoft.com/en-us/services/iot-edge/) infrastructure and all other components required to enable seamless communication to Azure. It significantly reduces the effort of building your own custom network server and deployment to multiple gateways. 
```

Comment:
(First sentence) For easier reading, I would add "," after the word implementation .. The Azure IoT Edge LoRaWAN starter kit is an *experimental* cross platform LoRaWAN implementation, following the [LoRaWAN 1.0.2 specification](https://lora-alliance.org/resource-hub/lorawantm-specification-v102).